src/components/TransactionFormInput.js

```diff
@@ -9,64 +9,77 @@ TransactionFormInput.propTypes = {
       onSubmit={(form, { resetForm }) => {
         onSave(form.value, form.notes, form.tag)
-        resetForm({ values: '' })
+        resetForm({
+          values: { value, notes, tag },
+        })
         history.push('/')
       }}
     >
-      {({ errors, touched }) => (
-        <FormStyled>
-          <Input name="value" placeholder="Enter transaction. Format: 20,00" />
-          {errors.value && touched.value ? (
-            <ErrorMessageInputStyled>{errors.value}</ErrorMessageInputStyled>
-          ) : null}
-          <LableStyled htmlFor="value">Euro</LableStyled>
-          <DropDown name="tag" component="select">
-            <option value="" selcted hidden>
-              -- Choose a tag --
-            </option>
-            <option value="Food">Food</option>
-            <option value="Leisure">Leisure</option>
-            <option value="FixedCosts">FixedCosts</option>
-            <option value="Transportation">Transportation</option>
-            <option value="Miscellaneous">Miscellaneous</option>
-            <option value="Earnings">Earnings</option>
-          </DropDown>
+      {({ values, errors, touched }) => {
+        return (
+          <FormStyled>
+            <Input
+              name="value"
+              placeholder="Enter transaction."
+              value={values.value}
+            />
+            {errors.value && touched.value ? (
+              <ErrorMessageInputStyled>{errors.value}</ErrorMessageInputStyled>
+            ) : null}
+            <LableStyled htmlFor="value">Euro</LableStyled>
+            <DropDown name="tag" component="select" value={values.tag}>
+              <option value="" hidden>
+                -- Choose a tag --
+              </option>
+              <option value="Food">Food</option>
+              <option value="Leisure">Leisure</option>
+              <option value="FixedCosts">FixedCosts</option>
+              <option value="Transportation">Transportation</option>
+              <option value="Miscellaneous">Miscellaneous</option>
+              <option value="Earnings">Earnings</option>
+            </DropDown>
 
-          {errors.tag && touched.tag ? (
-            <ErrorDropDownStyled>{errors.tag}</ErrorDropDownStyled>
-          ) : null}
-          <Notes
-            type="textarea"
-            name="notes"
-            placeholder="Add notes to transaction"
-          />
-          {errors.notes && touched.notes ? (
-            <ErrorMessageNotesStyled>{errors.notes}</ErrorMessageNotesStyled>
-          ) : null}
+            {errors.tag && touched.tag ? (
+              <ErrorDropDownStyled>{errors.tag}</ErrorDropDownStyled>
+            ) : null}
+            <Notes
+              type="textarea"
+              name="notes"
+              placeholder="Add notes to transaction"
+              value={values.notes}
+            />
+            {errors.notes && touched.notes ? (
+              <ErrorMessageNotesStyled>{errors.notes}</ErrorMessageNotesStyled>
```

Comment:
Could this be possible, too? Styled seems a little bit too much here
```suggestion
              <ErrorMessageNotes>{errors.notes}</ErrorMessageNotes>
```

Reply:
Good point